### PR TITLE
Institution / Credential Endpoints + Specs

### PR DESCRIPTION
**Pull request by vintagepenguin**
Adds institution endpoints. ⚡️ 

@ztoolson 
@jjcarstens

### CODE REVIEW

File: lib/atrium/institution.rb [hunk 1]
```diff
@@ -1,17 +1,37 @@
...
-    
-    ##
-    # GET /institutions
-    #
-    def self.list
-      ::Atrium.client.make_request(:get, "/institutions")
+
+    def self.list(options = nil)
+      endpoint = format_endpoint("institutions", options)
+      response = ::Atrium.client.make_request(:get, endpoint)
```
Comment (ztoolson) [inline lib/atrium/institution.rb]: So this list will only get the first page? Do we need to check if there are additional pages we need to fetch based on the response?

Comment (ztoolson) [inline lib/atrium/institution.rb]: Spoke with @vintagepenguin offline. pagination will come in a follow up PR.

**Comment by ztoolson** [comment]
:shipit:

File: lib/atrium/institution.rb (hunk 1 above)
Comment (vintagepenguin) [inline lib/atrium/institution.rb]: Adding pagination in a separate PR 👍

**Pull request merged**